Dockerfile

```diff
@@ -1,33 +1,11 @@
-FROM docs/docker.github.io:docs-base
+# Set to the version for this archive
+ARG VER=v17.06
 
-# docs-base contains: GitHub Pages, nginx, wget, svn, and the docs archives,
-# running on Alpine. See the contents of docs-base at:
-# https://github.com/docker/docker.github.io/tree/docs-base
+# This image comes from the Dockerfile.onbuild file in the docs-builder branch
+# https://github.com/docker/docker.github.io/blob/docs-builder/Dockerfile.onbuild
```

Comment:
This is now https://github.com/docker/docker.github.io/blob/publish-tools/Dockerfile.builder.onbuild